react-array-methods/src/components/MethodsListsContainer/index.tsx

```diff
@@ -0,0 +1,33 @@
+import MethodsList from '../MethodsList';
+import Tooltip from '../Tooltip';
+import IMethodsList from './IMethodsList';
+
+import './index.css';
+
+interface IMethodsListsContainerProps {
+  readonly: boolean;
```

Comment:
UpperCase I and Props in one time?
Here only?

Reply:
Not here only, I used upper-case I for all interfaces, regardless of whether they are Props or not